Source/WebKit/WebProcess/WebCoreSupport/WebPermissionController.cpp

```diff
@@ -103,8 +99,14 @@ void WebPermissionController::tryProcessingRequests()
             continue;
         }
 
+        auto* webPage = WebProcess::singleton().webPage(currentRequest.identifier);
+        if (!webPage) {
+            auto takenRequest = m_requests.takeFirst();
+            takenRequest.completionHandler(WebCore::PermissionState::Prompt);
```

Comment:
Can be written in one line: `m_requests.takeFirst().completionHandler(cachedResult);`

Reply:
We've gone back to using WebPageProxyIdentifier here and not checking the existence of the page.